Userland/Services/AudioServer/ClientConnection.cpp

```diff
@@ -149,4 +149,18 @@ void ClientConnection::set_main_mix_muted(bool muted)
 {
     m_mixer.set_muted(muted);
 }
+
+Messages::AudioServer::IsSelfMutedResponse ClientConnection::is_self_muted()
+{
+    if (m_queue)
+        return m_queue->is_muted();
+
+    return false;
+}
+
+void ClientConnection::set_self_muted(bool muted)
+{
+    if (m_queue)
+        m_queue->set_muted(muted);
```

Comment:
(2/2) You add `ClientAudioStream::set_muted` in the next commit, reorder these commits. :^)